scout/server/blueprints/variant/controllers.py

```diff
@@ -422,6 +422,52 @@ def variant_rank_scores(store, case_obj, variant_obj):
     return rank_score_results
 
 
+def get_loqusdb_obs_cases(store, variant_obj, category, obs_families=[]):
```

Comment:
I've extracted some code from the observations() function below to reduce its complexity. Created this new function with that code